packages/blip/test/unit/components/chart/trends.test.js

```diff
@@ -209,7 +209,7 @@ describe("Trends", () => {
       expect(wrapper.state()?.atMostRecent, "atMostRecent").to.be.true;
     });
 
-    it("should set the extendSize", async () => {
+    it.skip("should set the extendSize", async () => {
```

Comment:
You may have update the timeout, to something more, like 100ms ?

Reply:
No changelog ?

Reply:
I did not spent a lot of time trying to fix it as I cannot reproduce it locally. Maybe this would do the trick. For now, I just want the CI to stop randomly failing. There is a JIRA to fix it if you are interested https://diabeloop.atlassian.net/browse/YLP-1446 :slightly_smiling_face:
Indeed I did forget the changelog. I thought that this kind of modification were not relevant in it.